test/integration/testing/functional.test.js

```diff
@@ -145,3 +146,7 @@ testProp(
     t.is(typeof result, typeof stubResult);
   },
 );
+
+test("throwscape", (t) => {
+  t.throws(() => new Throwscape(options), { instanceOf: Error });
+});
```

Comment:
Upgrade this to a [generative test](https://github.com/ericcornelissen/shescape/blob/ced66ae3d908b76f5f56886be7e4d15ce888f853/CONTRIBUTING.md#generative-tests) for arbitrary `options`. (Also, this test is currently incorrect because `options` is undefined.)